api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayWork.scala

```diff
@@ -6,6 +6,8 @@ import org.elasticsearch.action.get.GetResponse
 import uk.ac.wellcome.models._
 import uk.ac.wellcome.utils.JsonUtil
 
+case class WorksIncludes(identifiers: Boolean = false)
```

Comment:
Two things I’m not sure about: is this the right place to define this class? And is this the right name?